### PR TITLE
Update aerosol climatology to 2013-2024 mean

### PR DESCRIPTION
**Pull request by AnningCheng-NOAA**
# Description
use the updated 2013 to 2024 mean MERRA2 climatology  instead of 2003 to 2014 mean

Depends on #2887 
Refs: ufs-community/ufs-weather-model#2272
Refs: ufs-community/ufs-weather-model#2273

# Type of change
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)?  NO
- Does this change require a documentation update? NO
- Does this change require an update to any of the following submodules? YES (If YES, please add a link to any PRs that are pending.)
  - [ ] EMC verif-global
  - [ ] GDAS
  - [ ] GFS-utils
  - [ ] GSI
  - [ ] GSI-monitor
  - [ ] GSI-utils
  - [ ] UFS-utils
  - [X] UFS-weather-model
  - [ ] wxflow

# How has this been tested?

# Checklist
- [X ] Any dependent changes have been merged and published
- [ X] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] My changes generate no new warnings
- [X ] New and existing tests pass with my changes
- [X ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by WalterKolczynski-NOAA — DISMISSED** [review]

**Comment by WalterKolczynski-NOAA** [comment]
Hold CI testing until #2887 is completed

**Comment by WalterKolczynski-NOAA** [comment]
Is a model update actually required? I don't think the changes here depend on the model regression tests being updated.

**Comment by AnningCheng-NOAA** [comment]
No change of source code in the model needed.

**Review by CatherineThomas-NOAA — CHANGES_REQUESTED** [review]
Once #2887 is addressed, this PR will need an update to point to the new fix/aer timestamp.  Otherwise, looks good.

**Review by NeilBarton-NOAA — DISMISSED** [review]

**Comment by WalterKolczynski-NOAA** [comment]
> Once #2887 is addressed, this PR will need an update to point to the new fix/aer timestamp. Otherwise, looks good.

#2887 is adding files, not replacing files, so no version increment should be needed.

**Comment by CatherineThomas-NOAA** [comment]
> https://github.com/NOAA-EMC/global-workflow/issues/2887 is adding files, not replacing files, so no version increment should be needed.

Good point @WalterKolczynski-NOAA.  Thanks.

**Review by CatherineThomas-NOAA — APPROVED** [review]

File: ush/forecast_predet.sh [hunk 1]
```diff
@@ -498,7 +498,7 @@ FV3_predet(){
     local month mm
     for (( month = 1; month <=12; month++ )); do
       mm=$(printf %02d "${month}")
-      ${NCP} "${FIXgfs}/aer/merra2.aerclim.2003-2014.m${mm}.nc" "aeroclim.m${mm}.nc"
+      ${NCP} "${FIXgfs}/aer/merra2_1423_${mm}.nc" "aeroclim.m${mm}.nc"
```
Comment (aerorahul) [inline ush/forecast_predet.sh]: Can these files be named `merra2.aerclim.2014-2023.m${mm}.nc`?

Comment (WalterKolczynski-NOAA) [reply ush/forecast_predet.sh]: I took care of the file renaming when staging the new fix files. Just this line needs to be changed now.
```suggestion
      ${NCP} "${FIXgfs}/aer/merra2.aerclim.2014-2023.m${mm}.nc" "aeroclim.m${mm}.nc"
```

**Comment by AnningCheng-NOAA** [comment]
sure. Please make the changes accordingly.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**